pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1862,32 +1848,8 @@ func getDefaultNetworkTags(ctx context.Context, client client.Client, customTag
 	return tags, nil
 }
 
-// resources such as cluster vpc route tables are tagged with a cluster owner tag
-// denoted -> ` kubernetes.io/cluster/<cluster-id>=owned
-// this utility function builds and returns the tag
-func getClusterOwnerTag(ctx context.Context, client client.Client) (*ec2.Tag, error) {
-	return getClusterTag(ctx, client, clusterOwnedTagValue)
-}
-
-func getClusterSharedTag(ctx context.Context, client client.Client) (*ec2.Tag, error) {
-	return getClusterTag(ctx, client, clusterSharedTagValue)
-}
-
-func getClusterTag(ctx context.Context, client client.Client, tagValue string) (*ec2.Tag, error) {
-	clusterID, err := resources.GetClusterID(ctx, client)
-	if err != nil {
-		return nil, errorUtil.Wrap(err, "error getting clusterID")
-	}
-
-	// a tag for identifying cluster owned vpcs
-	return &ec2.Tag{
-		Key:   aws.String(getOSDClusterTagKey(clusterID)),
-		Value: aws.String(tagValue),
-	}, nil
-}
-
```

Comment:
Not used, cleaning up

Reply:
@cecobask this functionality was added to support byovpc. Could you point to where the cluster vpc is gotten now? If it's moved that's fine but we still need to include both `owned` and `shared` otherwise we'll remove support for byovpc.

Reply:
I'm unsure when the references were removed but there are no uses of these functions on current master version - or at least that's what my linter tells me.

Reply:
@adam-cattermole +1
@laurafitzgerald Not sure how byovpc works, but I believe the function was abandoned in this PR: https://github.com/integr8ly/cloud-resource-operator/pull/244/files

Reply:
The pr you linked is where the shared tag was added. 🤔  https://github.com/integr8ly/cloud-resource-operator/pull/244/files#diff-691829d67b270bfdcac4bcee0b5a0c2517c3885fa20c7d78cb27703ed6136ac5R60

Reply:
Hi @cecobask sorry for the delay. I was thinking of the shared tag here-> https://github.com/laurafitzgerald/cloud-resource-operator/blob/852d9dd0d43b2fa74a82cb431208e692f88d3087/pkg/providers/aws/cluster_vpc.go#L468 so we can remove these functions.